pipeline/acquisition.py

```diff
@@ -10,9 +10,9 @@
 import datajoint as dj
 import h5py as h5
 
-from . import reference, subject, utilities, stimulation
+from . import reference, subject, utilities
 
-schema = dj.schema(dj.config.get('database.prefix', '') + 'gi2017_acquisition')
+schema = dj.schema(dj.config.get('database.prefix', '') + 'acquisition')
```

Comment:
Shall we go ahead and place all custom config under `dj.config['custom']` after all?